src/components/datepicker/OcDatepicker.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <date-picker class="oc-datepicker" v-bind="$attrs" v-on="$listeners">
+    <template #default="{ inputValue, togglePopover, hidePopover }">
+      <!-- @slot Default slot to use as the popover anchor for datepicker -->
+      <slot :inputValue="inputValue" :togglePopover="togglePopover" :hidePopover="hidePopover" />
+    </template>
+  </date-picker>
+</template>
+
+<script>
+import DatePicker from "v-calendar/lib/components/date-picker.umd"
+
+/**
+ * Datepicker component based on [v-calendar](https://vcalendar.io/)
```

Comment:
Could you make it a little bit more obvious that the documentation for the props is external? Like you already did with OcSelect? There it is this text: `For detailed documentation (props, slots, events, etc.), please visit https://vue-select.org/`

Reply:
@kulmann added it, please review and feel free to merge this PR